### PR TITLE
[DO NOT MERGE] test absolute paths

### PR DESCRIPTION
**Pull request by autoantwort**
Test run for https://github.com/microsoft/vcpkg-tool/pull/172

### CODE REVIEW

**Review by autoantwort — COMMENTED** [review]
.

**Review by VerwaltungHilton — CHANGES_REQUESTED** [review]
Please do that

File: scripts/azure-pipelines/azure-pipelines.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ parameters:
   - name: vcpkgToolSha
     displayName: 'Custom SHA of vcpkg-tool to use rather than bootstrap'
     type: string
-    default: 'use default'
+    default: 'cc3f741c05092f7938f150872d677e26bd36f71a'
```
Comment (VerwaltungHilton) [inline scripts/azure-pipelines/azure-pipelines.yml:13]:
```suggestion
    default: 'cc3f741c05092f798f150872d677e26bd36f71a'

```

**Review by VerwaltungHilton — CHANGES_REQUESTED** [review]
test

**Review by VerwaltungHilton — DISMISSED** [review]

**Comment by BillyONeal** [comment]
As the PR has been merged into the tool I think we can close this now?

**Comment by autoantwort** [comment]
It is merged into the tool but not into the vcpkg repo, so people can still add absolute paths currently (and they do about twice a week). And I want a full successful run so that there are no regressions when the new tool gets merged

**Pull request closed without merging**